napari/_qt/preferences_schema.py

```diff
@@ -0,0 +1,70 @@
+from json import dumps
+
+from qtpy import QtWidgets
+
+from napari._vendor.qt_json_builder.qt_jsonschema_form import WidgetBuilder
+
+
```

Comment:
Should this file live in _qt/dialogs ?  I'm not sure it belongs where I have it, but that's not been mentioned or noticed yet.

Reply:
nah... if it's clearly being vendored, leave it in its vendored location and import from there.  If you need to modify it significantly, you can create a subclass outside of vendored.  Particularly in this case since `WidgetBuilder` isn't actually a `QWidget itself`, it shouldn't go in `_qt/dialogs`

Reply:
thanks @tlambert03 for the reply! Sorry, I was really confusing.  I meant to ask about the file `preferences_schema.py`.  I have it in /_qt, but this is the code that is used to create the dialog from the vendored code.

Reply:
and when I say confusing, I mean...not clear at all.  so sorry!

Reply:
oh, not your fault.  my bad... I always get confused about looking above or below the comment :joy:.  In that case, I'd say yes!  If the primary export from this module is a dialog, it would fit well in `_qt/dialogs.py`  thanks!

Reply:
thank you!  I will move it!